SIS_import/sis_set_url.rb

```diff
@@ -523,14 +525,18 @@ def read_argv
 
 			esbToken=refreshESBToken()
 
-			# update MPathway with Canvas urls
-			updateError = update_MPathway_with_Canvas_url(esbToken, outputDirectory)
+			if (!esbToken.nil?)
+				# update MPathway with Canvas urls
+				updateError = update_MPathway_with_Canvas_url(esbToken, outputDirectory)
```

Comment:
the updateError variable is used to carry all possible errors, including script setting error, ESB connection error, and Canvas call returned error. A better naming may be needed here. But for this emergency release, I would like to keep the changes as few as possible.